File: lib/models/campaign/campaign.dart
class Campaign {
  int campaignId;
  String campaignName;
  String firstName;
  String lastName;
  int careless;
  String startDate;
  String endDate;
  String description;
  String image;
  double amount;
  double currentlyMoney;
  int categoryId;
  int userId;
  Campaign(
      {this.campaignId,
      this.campaignName,
      this.description,
      this.careless,
      this.endDate,
      this.firstName,
      this.lastName,
      this.startDate,
      this.amount,
      this.currentlyMoney,
      this.categoryId,
      this.userId,
      this.image});

  factory Campaign.fromJson(Map<String, dynamic> json) {
    return Campaign(
      campaignId: json['campaignId'],
      campaignName: json['campaignName'],
      description: json['description'],
      careless: json['careless'],
      firstName: json['firstName'],
      lastName: json['lastName'],
      startDate: json['startDate'],
      endDate: json['endDate'],
      amount: json["amount"],
      currentlyMoney: json["currentlyMoney"],
      image: json['image'],
    );
  }

  Map toJson() => {
        'campaignId': campaignId,
        'campaignName': campaignName,
        'description': description,
        'careless': careless,
        'firstName': firstName,
        'lastName': lastName,
        'startDate': startDate,
        'endDate': endDate,
        'amount': amount,
        'currentlyMoney': currentlyMoney,
        'image': image,
        'userId': userId,
        'categoryId': categoryId
      };
}

File: SWD/Models/Campaign.cs
using System;
using System.Collections.Generic;

namespace SWD.Models
{
    public partial class Campaign
    {
        public Campaign()
        {
            DonateDetail = new HashSet<DonateDetail>();
            LikeDetail = new HashSet<LikeDetail>();
        }

        public string CampaignName { get; set; }
        public double Amount { get; set; }
        public double CurrentlyMoney { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Description { get; set; }
        public bool Approved { get; set; }
        public bool? IsDeleted { get; set; }
        public int? CategoryId { get; set; }
        public int? UserId { get; set; }
        public int Id { get; set; }
        public int? Popular { get; set; }
        public string Image { get; set; }

        public Category Category { get; set; }
        public User User { get; set; }
        public ICollection<DonateDetail> DonateDetail { get; set; }
        public ICollection<LikeDetail> LikeDetail { get; set; }
    }
}
